sympy/vector/operators.py

```diff
@@ -0,0 +1,184 @@
+from sympy.core import Expr
+from sympy.vector.coordsysrect import CoordSysCartesian
+from sympy.vector.vector import Vector
+from sympy.vector.deloperator import Del
+
+
+class Gradient(Expr):
+    """
+    Represents unevaluated Gradient.
+
+    Examples
+    ========
+
+    >>> from sympy.vector import CoordSysCartesian, Gradient
+    >>> R = CoordSysCartesian('R')
+    >>> s = R.x*R.y*R.z
+    >>> Gradient(s, R)
+    Gradient(R.x*R.y*R.z, R)
+
+    """
+
+    def __new__(cls, scalar, coord_sys):
```

Comment:
Let's remove the `coord_sys` dependency, let's define the operators as close as possible to how they are defined in the math books.